stratum/hal/lib/tdi/tofino/tofino_switch.h

```diff
@@ -108,14 +102,6 @@ class TofinoSwitch : public SwitchInterface {
   // invalid/unknown/uninitialized node.
   ::util::StatusOr<TdiNode*> GetTdiNodeFromNodeId(uint64 node_id) const;
 
-  // Pointer to a PhalInterface implementation. The pointer has been also
-  // passed to a few managers for accessing HW. Note that there is only one
-  // instance of this class per chassis.
-  PhalInterface* phal_interface_;  // not owned by this class.
```

Comment:
What was PhalInterface for?

Why don't we need `TdiSdeInterface` anymore ? How would we make PD-fixed calls if required?

Reply:
PhalInterface is platform HAL to manager SFP insertion and removal. It is entirely used within chassis manager. So moved its init and deinit within there.
Again TdiSdeInterface has no use here and it should not. Any ASIC related call has to go through tdi_node or tofino_chassis_manager. PD-fixed today is entirely in chassis manager.